.github/workflows/ci.yml

```diff
@@ -45,15 +45,12 @@ jobs:
         env:
           CI: true
 
-      - name: lint
-        run: |
-          yarn lint
-          yarn workspace stats lint
-
       - name: build sdk packages
         run: |
           yarn workspace @growthbook/growthbook build
           yarn workspace @growthbook/growthbook-react build
+          yarn lint
+          yarn workspace stats lint
```

Comment:
Without the SDK's having been built, the lint tasks fail. See previous failed builds for errors.